### DISCUSSION TITLE
v2.33.0

### DISCUSSION THREAD

**Discussion by nabokihms** [dexidp/dex#2608] [Announcements]
### Exciting New Features 🎉
* add PKCE support to device code flow by @bobcallaway in https://github.com/dexidp/dex/pull/2575
### Enhancements 🚀
* Limit the amount of objects we attempt to GC on each cycle by @kellyma2 in https://github.com/dexidp/dex/pull/2524
* Use GitLab's refresh_token during Refresh. by @dhaus67 in https://github.com/dexidp/dex/pull/2352
* Add domainHint parameter to Microsoft Connector by @josephtknight in https://github.com/dexidp/dex/pull/2586
* add config to explicitly set scopes for microsoft connector by @bobcallaway in https://github.com/dexidp/dex/pull/2582
### Bug Fixes 🐛
* fix: prevent cross-site scripting for the device flow by @nabokihms in https://github.com/dexidp/dex/pull/2468
* grpc-client: Do not crash on empty response by @bbusse in https://github.com/dexidp/dex/pull/2584
### Dependency Updates ⬆️
* build(deps): bump helm/kind-action from 1.2.0 to 1.3.0 by @dependabot in https://github.com/dexidp/dex/pull/2555
* build(deps): bump aquasecurity/trivy-action from 0.3.0 to 0.4.0 by @dependabot in https://github.com/dexidp/dex/pull/2557
* build(deps): bump github.com/stretchr/testify from 1.7.2 to 1.8.0 by @dependabot in https://github.com/dexidp/dex/pull/2577
* build(deps): bump aquasecurity/trivy-action from 0.4.0 to 0.5.1 by @dependabot in https://github.com/dexidp/dex/pull/2576
* build(deps): bump mheap/github-action-required-labels from 1 to 2 by @dependabot in https://github.com/dexidp/dex/pull/2565
* build(deps): bump google.golang.org/api from 0.82.0 to 0.86.0 by @dependabot in https://github.com/dexidp/dex/pull/2574
* build(deps): bump github.com/spf13/cobra from 1.4.0 to 1.5.0 by @dependabot in https://github.com/dexidp/dex/pull/2560
* build(deps): bump aquasecurity/trivy-action from 0.5.1 to 0.6.0 by @dependabot in https://github.com/dexidp/dex/pull/2602
* build(deps): bump alpine from 3.16.0 to 3.16.1 by @dependabot in https://github.com/dexidp/dex/pull/2598
* build(deps): bump golang from 1.18.3-alpine3.15 to 1.18.4-alpine3.15 by @dependabot in https://github.com/dexidp/dex/pull/2592
* build(deps): bump github.com/sirupsen/logrus from 1.8.1 to 1.9.0 by @dependabot in https://github.com/dexidp/dex/pull/2599
* build(deps): bump github.com/go-ldap/ldap/v3 from 3.4.2 to 3.4.4 by @dependabot in https://github.com/dexidp/dex/pull/2606
* build(deps): bump google.golang.org/api from 0.86.0 to 0.89.0 by @dependabot in https://github.com/dexidp/dex/pull/2605
* build(deps): bump aquasecurity/trivy-action from 0.6.0 to 0.6.1 by @dependabot in https://github.com/dexidp/dex/pull/2604

## New Contributors
* @kellyma2 made their first contribution in https://github.com/dexidp/dex/pull/2524
* @josephtknight made their first contribution in https://github.com/dexidp/dex/pull/2586
* @bbusse made their first contribution in https://github.com/dexidp/dex/pull/2584

**Full Changelog**: https://github.com/dexidp/dex/compare/v2.32.0...v2.33.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dexidp/dex/releases/tag/v2.33.0'>v2.33.0</a>.</em>

**Comment by nunojusto** [comment]
Hi, no container image release?

**Reply by sagikazarmark** [reply]
You can find the latest version here: https://github.com/dexidp/dex/pkgs/container/dex

Updated the release description as well

**Reply by nunojusto** [reply]
Thank you
